### PR TITLE
Fix Navbar button in mobile view

### PR DESCRIPTION
**Pull request by Bawanthathilan**
## Purpose

Navbar button  displaying in mobile view #298 

## Approach
add custom CSS in global-custom.css file

### Screenshots
image

 ## Preview Link
https://bawantharathnayakasliit.github.io/sef-site/src/main/webapp/

### CODE REVIEW

File: src/main/webapp/assets/css/global-custom.css [hunk 1]
```diff
@@ -313,4 +313,10 @@ body{
     border-color: #CEA4F5;
     background-color: #CEA4F5;
     box-shadow: 0 4px 6px rgba(50, 50, 93, .11), 0 1px 3px rgba(0, 0, 0, .08);
+}
+
+.navbar-toggler-icon
+{
+
```
Comment (jayasanka-sack) [inline src/main/webapp/assets/css/global-custom.css]: Remove unwanted empty line

File: src/main/webapp/assets/css/global-custom.css [hunk 2]
```diff
@@ -313,4 +313,10 @@ body{
     border-color: #CEA4F5;
     background-color: #CEA4F5;
     box-shadow: 0 4px 6px rgba(50, 50, 93, .11), 0 1px 3px rgba(0, 0, 0, .08);
+}
+
+.navbar-toggler-icon
+{
```
Comment (jayasanka-sack) [inline src/main/webapp/assets/css/global-custom.css]: Move this curly brace to upper line. 
`.navbar-toggler-icon {`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/assets/css/global-custom.css (hunk 2 above)
Comment (Bawanthathilan) [reply src/main/webapp/assets/css/global-custom.css]: https://github.com/sef-global/sef-site/pull/309

**Comment by jayasanka-sack** [comment]
Thanks @bawantharathnayakasliit !

**Pull request merged**